logstash-input-tcp.gemspec

```diff
@@ -1,13 +1,14 @@
 Gem::Specification.new do |s|
   s.name          = 'logstash-input-tcp'
-  s.version         = '4.1.2'
+  s.version       = ::File.read('version').split("\n").first
```

Comment:
@jordansissel I already did it the same way in the user agent input filter , the motivation was just to not duplicate the version between Gemspec and Gradle build file.
Not cool? :)